create_aio_app/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/app.py

```diff
@@ -3,16 +3,19 @@
 
 {%- if cookiecutter.use_redis == 'y' %}
 from functools import partial
+import aioredis
```

Comment:
This change is unrelated. And also this import doesn't belong to the section with stdlib imports.

Reply:
Got it.  Reverted it back.